helm/values.yaml

```diff
@@ -25,6 +25,7 @@ coder:
   # coder.annotations -- The Deployment annotations. See:
   # https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
   annotations: {}
```

Comment:
We should probably rename `annotations` to `deployment_annotations` then and deprecate `annotations`.

If you aren't comfortable with doing that just let us know and we'll take it over!

Reply:
@kylecarbs Hi, I'm happy to contribute. I will rename it :)

Reply:
@kylecarbs wdyt?